README.md

```diff
@@ -23,3 +23,61 @@ later), this module simply reexports `Quote` and `Code`
 from `Language.Haskell.TH.Syntax`. Refer to the Haddocks
 for `Language.Haskell.TH.Syntax.Compat` for examples of
 how to use this module.
+
+## Quick Start Guide
+
+Let's say you have a library that offers a `foo :: Q (TExp a)`
+and you want to make it compatible across this version.
```

Comment:
I would add an additional requirement to this sentence to the effect of "and you intend for users to splice `foo` directly as a Template Haskell expression". I'm choosing to be picky about this since, strictly speaking, it's not the only way to approach the issue of backwards-compatibility—you could also choose to make the API use `Code` uniformly using this library and use the `IsCode` class to mediate splicing it in. Whether you prefer `Splice` versus `IsCode` is a matter of requirements, so better to state this use case's requirements up front.